packages/frontend/src/components/Trade/Long/index.tsx

```diff
@@ -960,11 +985,31 @@ type BuyProps = {
   activeStep?: number
 }
 
+type LongProps = {
+  sqthBalProps: {
+    longSqthBal: BigNumber
+    refetch: () => void
+    loading: boolean
+    error?: any
+  }
+}
+
 const Long: React.FC<BuyProps> = ({ open, isLPage = false, activeStep = 0 }) => {
+  const { longSqthBal, refetch: refetchSqthBal, loading: loadingSqthBal, error } = useLongSqthBal()
+
   return open ? (
-    <OpenLong open={open} isLPage={isLPage} activeStep={activeStep} />
+    <OpenLong
+      open={open}
+      isLPage={isLPage}
+      activeStep={activeStep}
+      sqthBalProps={{ longSqthBal, refetch: refetchSqthBal, loading: loadingSqthBal }}
```

Comment:
I noticed there are some inconsistencies for long balance in the codebase, cuz there is also long balance in the osqth input of <OpenLong /> component, but the balance will change right after the open long trade finished, it gets updated with the same value on position card and at the same time. 

We dont have balance not loaded issue on buy side, cuz we r using [squeethAmount variable ](https://github.com/opynfinance/squeeth-monorepo/pull/385/files#diff-0df6f866c3e8aa6fd371a3fdb870c356e099f912847729a6c7693c57597901fcR511) as our long balance there, same as the value on position card and position page, but on long close, now we r using useLongSqthBal, which does some extra fetching and this hook is only used in long trade card, but think squeethAmount already did the fetching job to get correct position value. So we dont need extra fetching in long close component

So thinking it's better to have a jotai atom for long position value across the app, or at least we all use same value / variable for same position